pingPong/schema.sql

```diff
@@ -0,0 +1,13 @@
+create table users(
+    id serial primary key,
+    name text
+);
+
+create table games(
+    id serial primary key,
+    player1 text, 
+    player2 text,
+    winner text,
```

Comment:
I wonder if you might also want to have `player1_score` and `player2_score` fields? That might let you make the `winner` field a foreign key to the player with the higher score.